packages/panzer/adapters-stk/test/evaluator_tests/basis_times_vector.cpp

```diff
@@ -234,12 +234,14 @@ namespace panzer {
 
     auto fieldData_qedge1_h = Kokkos::create_mirror_view(fieldData_qedge1.get_view());
     Kokkos::deep_copy(fieldData_qedge1_h, fieldData_qedge1.get_view());
-    TEST_FLOATING_EQUALITY(fieldData_qedge1_h(0,0),5.0/12.0,1e-5);        // 0 edge basis is [(1-y_ref)/4, 0] 
-    TEST_FLOATING_EQUALITY(fieldData_qedge1_h(0,2),3.0/4.0,1e-5);         // 2 edge basis is [(1+y_ref)/4, 0] 
+
+    //
+    TEST_FLOATING_EQUALITY(fieldData_qedge1_h(0,0),5.0/6.0,1e-5);        // 0 edge basis is [(1-y_ref)/4, 0]
```

Comment:
Are the comments about the edge bases still correct, or are they off by a factor of 2 thanks to the change in basis definition?

Reply:
Good catch. I'll fix the comments.